### ISSUE TITLE
Build fails in 2.12

### ISSUE THREAD

**Issue by avargas-clorian** [MatthewHerbst/react-to-print#328]
There is a build error when trying to use v2.12
And an error when trying to install the library

` npm i -S react-to-print`

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/react-to-date - Not found
npm ERR! 404 
npm ERR! 404  'react-to-date@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

**Comment by MatthewHerbst** [comment]
Hello. This is the repo for the [`react-to-print`](https://www.npmjs.com/package/react-to-print)  library.

**Comment by avargas-clorian** [comment]
Yes, sorry misstype. I want to say react-to-print.

Install is ok.

 My builds with mui-datatables is failing due to react-to-print.


> react-scripts build

Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

        ./node_modules/react-to-print/lib/index.js:2:358

**Comment by MatthewHerbst** [comment]
Can you provide the full build process please? I just created an empty npm project locally and was able to install `mui-datatables` without issue

**Comment by avargas-clorian** [comment]
Maybe it's a problem with an old version of react-scripts (v1.1.4)
Look at the dependencies in my package.json. It starts to fail since v2.12 

```json
"dependencies": {
    "@material-ui/core": "^4.7.1",
    "@material-ui/icons": "^4.5.1",
    "bootstrap": "3.3.7",
    "env-cmd": "8.0.2",
    "i18n-iso-countries": "^3.7.8",
    "iban": "0.0.11",
    "js-cookie": "2.2.0",
    "jw-react-pagination": "^1.0.7",
    "jwt-decode": "^2.2.0",
    "material-ui-icons": "^1.0.0-beta.36",
    "moment": "2.20.1",
    "mui-datatables": "^2.13.1",
    "node-sass-chokidar": "1.3.3",
    "react": "^16.8.3",
    "react-bootstrap": "0.32.1",
    "react-bootstrap-tabs": "^1.0.2",
    "react-copy-to-clipboard": "^5.0.1",
    "react-csv": "1.0.8",
    "react-dates": "^17.2.0",
    "react-dom": "^16.8.3",
    "react-intl": "^2.8.0",
    "react-intl-translations-manager": "5.0.1",
    "react-loading": "^2.0.3",
    "react-notifications": "^1.4.3",
    "react-redux": "5.0.6",
    "react-router-dom": "4.3.1",
    "react-scripts": "1.1.4",
    "react-select": "^2.0.0",
    "react-sidenav": "^2.1.3",
    "react-slide-toggle": "^0.1.13",
    "react-tippy": "^1.2.2",
    "redux": "3.7.2",
    "redux-thunk": "2.2.0",
    "whatwg-fetch": "2.0.3"
  },
  "devDependencies": {
    "babel-cli": "^6.26.0",
    "babel-plugin-import-inspector": "^2.0.0",
    "babel-plugin-react-intl": "^2.4.0",
    "prettier": "^1.12.1"
  },
```

I think it's time to upgrade my project. Thank you anyway

**Comment by MatthewHerbst** [comment]
Please let me know if upgrading fixes the problem

**Comment by charles4221** [comment]
I am having this same issue. The problem is because after switching your minifier from Uglify to Terser in v2.12.0, the new minified bundle still contains untranspiled arrow functions which were previously transpiled to ES5 in the npm distributable for this library. Apps using create-react-app prior to v2.0.3 are unable to build because they cannot transpile node_modules.

**Comment by avargas-clorian** [comment]
After react-scripts upgrade the build is working fine.
Installing react-to-print with v2.11 also works. Even if react-scripts has version 1 it builds.

Thanks

**Comment by MatthewHerbst** [comment]
@charles4221 that doesn't exactly add up: the minifier doesn't transform code, that's what we use Typescript for. I did recently upgrade the Typescript version from 3 -> 4 though, so that could be the issue.. Investigating, thanks for letting me know the arrow functions are in the build, they for sure shouldn't be

**Comment by MatthewHerbst** [comment]
Problem found, it's an issue with the Webpack upgrade: https://webpack.js.org/blog/2020-10-10-webpack-5-release/#improved-code-generation

Pushing a fix in a few minutes

**Comment by MatthewHerbst** [comment]
Published as `2.12.1`. Please let me know if you run in any additional problems, and thank you greatly for reporting the issue!
